templates/unbound.modules.conf.erb

```diff
@@ -1,5 +1,3 @@
-# Managed by Puppet
```

Comment:
This should probably stay.

Reply:
that is concat into the unbound.conf that already states that at the beginning . Basically that is a duplicate